alphagov/gds-way
# Service Level Objectives

### What
- Added Service Level Objectives section to Logging and Monitoring
- Added an SLO burn image

### How to review
- pull the code and check it runs
- Check the content and spot any errors in the writing
- Check the image link.
